_layouts/platform.html

```diff
@@ -65,8 +65,10 @@ <h2 class="d-inline h1">{{ page.subtitle }}</h2>
           {% assign platform = split[2] %}
 
           <div class="col-12 col-lg-4">
+            {% if page.hide_activities != true %}
```

Comment:
```suggestion
            {% unless page.hide_activities %}
```

I'd use `unless` because I think a missing `hide_activity` in the platform file would be falsy. (not sure though)